examples/dialogs/dialogs/app.py

```diff
@@ -66,6 +81,16 @@ def action_select_folder_dialog(self, widget):
             self.label.text = "Folder selected:" + ','.join([path for path in path_names])
         except ValueError:
             self.label.text = "Folder select dialog was canceled"
+            
+    def action_select_folder_dialog_with_initial_folder(self, widget):
+        try:
+            path_names = self.main_window.select_folder_dialog(
+                title="Select folder with Toga in home folder",
+                initial_directory='./..'
```

Comment:
This can't work; relative paths can't be expressed as a URI. 

The best option I can think of is to use `str(Path.home())` - it's a very roundabout way of getting the job done, but it provides a guaranteed valid path, and does it as a string.